src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Ball ball = new Ball();
+
+    public Ball getRandomBall() {
+        ball.setNumber(random.nextInt(BOUND));
+        ball.setColor(colorSupplier.getRandomColor());
+        return ball;
+    }

```

Comment:
we should create new ball each time `getRandomBall` was called (but now we just change the same object)

also it would be better to have a constructor for `Ball` class (like `public Ball(String color, int number)`), 
so we could create our balls with parameters that we need (like `new Ball(someColor, someNumber)`)